ingestion/src/metadata/utils/helpers.py

```diff
@@ -123,7 +123,7 @@ def get_start_and_end(duration: int = 0) -> Tuple[datetime, datetime]:
     Method to return start and end time based on duration
     """
 
-    today = datetime.now(timezone.utc)
+    today = datetime.now(timezone.utc).replace(tzinfo=None)
```

Comment:
why pass the timezone and then remove it instead of `datetime.now()` without any specification?

Reply:
just using `datetime.now()` will give the local time which can be different from the UTC time. to keep it standard we pass the timezone as `utc`